### PR TITLE
Version 2.0

### PR DESCRIPTION
**Pull request by mark-jensen**
Updated namespace and switch to opaque IDs.

### CODE REVIEW

**Comment by mark-jensen** [comment]
This still requires documentation, but the ttl files are ready for review. 
@neilotte @johnbeve @APCox 

Also note the format of MRO IRIs and `Modal Object Property`

**Review by neilotte — APPROVED** [review]
@mark-jensen Looks good. Something I did see (not a showstopper. We can look at in 2.1) is that asWKT is declared in Geospatial Ontology, while no other third-party property presently is. Don't know if that is intentional or not.

**Review by johnbeve — APPROVED** [review]

File: documentation/mappping-new-iris/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# mapping NEw IRIs
+
+This directory contains the mapping file for renaming the version 1.7 and older CCO ontology elements to the schema intorduced in version 2.0
```
Comment (APCox) [inline documentation/mappping-new-iris/README.md]: correct misspelling of 'introduced

File: documentation/mappping-new-iris/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+It is desiging to work with the ROBOT command `rename`
```
Comment (APCox) [inline documentation/mappping-new-iris/README.md]: correct misspelling of 'designed'

File: documentation/mappping-new-iris/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+robot rename -i [file-name-old-iris] --mappings iri-mapping-v2.0.csv --add-prefix "cco2: https://www.commoncoreontologies.org/" --allow-missing-entities true -o [file-name-new-iris]
+
+Ontology IRIs require a manual find and replace.
+
```
Comment (APCox) [inline documentation/mappping-new-iris/README.md]: Should All Core Ontology be added to this list? It has no content, but it does have an IRI that changed.

File: src/cco-merged/AllCoreOntology.ttl [hunk 4]
```diff
@@ -1,26 +1,26 @@
-@prefix : <http://www.ontologyrepository.com/CommonCoreOntologies/> .
-@prefix cco: <http://www.ontologyrepository.com/CommonCoreOntologies/> .
+@prefix : <https://www.commoncoreontologies.org/AllCoreOntology/> .
```
Comment (APCox) [inline src/cco-merged/AllCoreOntology.ttl:1]: 'AllCoreOntology/' should be deleted from this line

File: src/cco-merged/AllCoreOntology.ttl [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
+@prefix cco: <https://www.commoncoreontologies.org/AllCoreOntology> .
```
Comment (APCox) [inline src/cco-merged/AllCoreOntology.ttl:2]: 'AllCoreOntology/' should definitely be deleted from this line

File: documentation/mappping-new-iris/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ArtifactOntology --> https://www.commoncoreontologies.org/ArtifactOntology
+http://www.ontologyrepository.com/CommonCoreOntologies/Mid/AgentOntology --> https://www.commoncoreontologies.org/AgentOntology
+http://www.ontologyrepository.com/CommonCoreOntologies/Mid/InformationContentOntology --> https://www.commoncoreontologies.org/InformationContentOntology
```
[the remark this answers is not in the document]
Comment (mark-jensen) [reply documentation/mappping-new-iris/README.md]: @APCox  you are commenting on an earlier version of this file. These fixes were made 6 hours ago as part of b9b25d7554326c09842ec2c4c39b6d2b4f00bd05

**Comment by mark-jensen** [comment]
@neilotte agree about asWKT.

**Pull request merged**